docs/operate/reference/components/button/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Button Component"
+linkTitle: "Button"
+childTitleEndOverwrite: "Button Component"
+weight: 10
+type: "docs"
+description: "Represents a physical button on a machine."
```

Comment:
I think this is more in line with descriptions for other components. We could consider adding "represents" to other ones but I'm not sure whether we do want that
```suggestion
description: "A physical button on a machine."
```